Parsers/measurement_parser.py

```diff
@@ -71,27 +71,25 @@ def get_measurements(ingredient):
     counter = 0
 
     while counter < len(ingredient_split):
-        current_ingredient = ingredient_split[counter]
-        if (current_ingredient.isnumeric() or (current_ingredient[0].isnumeric() and '/' in current_ingredient and len(current_ingredient) == 3)) and counter < len(ingredient_split) - 1:
-            ingredient_amount = eval(current_ingredient)
+        if (ingredient_split[counter].isnumeric() or (ingredient_split[counter][0].isnumeric() and '/' in ingredient_split[counter] and len(ingredient_split[counter]) == 3)) and counter < len(ingredient_split) - 1:
```

Comment:
over ehre you call ingredient_split[counter] 5 times in 2 lines, so this could be changed into a variable to simplifiy repetition and any potential debugging in the future

Reply:
and the same for all the other places it's called

Reply:
Switched to variable current_ingredient